src/GravityFormsExtensions.php

```diff
@@ -261,12 +261,12 @@ public function p4_gf_confirmation_settings(array $fields): array
      *
      * Return custom confirmation message for Gravity Forms.
      *
-     * @param string $confirmation The default confirmation message.
-     * @param array  $form The form properties.
+     * @param string|array $confirmation The default confirmation message.
+     * @param mixed        $form         The form properties.
      *
      * @return string The custom confirmation message.
      */
-    public function p4_gf_custom_confirmation(string $confirmation, array $form): string
+    public function p4_gf_custom_confirmation($confirmation, $form): string
```

Comment:
Hi @lithrel , The function return type will be string or array, we early return If its array. 🙂

Reply:
Damn :grimacing:  new PR then